### ISSUE TITLE
Feature Request | Show Speed when Replaying

### ISSUE THREAD

**Issue by Kit4444** [traccar/traccar-web#1214]
A wonderful day, evening, night to you,

I'd like to see the speed, the vehicle was going in Replay Mode.

The reason why is, you can see clearly, that you are going faster, as the points are further away from each other, however that is not really a way to "calculate" the actual going speed.

The solution should be pretty simple: When the tracker sends a bunch of new data, the server process it and also saves the current speed, why not just show it too? So we don't have to export data every day to determine, who drives according to law and who does not.

Kind Regards,
Christopher Wilhelm H.

**Comment by tananaev** [comment]
We do show the speed and any other data you want in the popup.

**Comment by Kit4444** [comment]
Yeah, I must be blind then - I'm sorry, but thanks for informing me! 😅
